src/main/java/wooteco/subway/web/member/favorite/FavoriteController.java

```diff
@@ -0,0 +1,61 @@
+package wooteco.subway.web.member.favorite;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.domain.member.Favorite;
+import wooteco.subway.domain.member.FavoriteDetail;
+import wooteco.subway.domain.member.Member;
+import wooteco.subway.service.member.favorite.FavoriteService;
+import wooteco.subway.service.member.favorite.dto.AddFavoriteRequest;
+import wooteco.subway.service.member.favorite.dto.FavoriteResponse;
+import wooteco.subway.service.member.favorite.dto.FavoritesResponse;
+import wooteco.subway.web.member.LoginMember;
+
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping("/members/{id}/favorites")
+public class FavoriteController {
+	private FavoriteService favoriteService;
```

Comment:
```suggestion
	private final FavoriteService favoriteService;
```
final을 붙여서 꼭 필요한 멤버변수라 표시해주세요

Reply:
final로 컨트롤러에서 꼭 필요한 서비스임을 명시해줬습니다🙂